docs/source/cuml_intro.rst

```diff
@@ -1,36 +1,57 @@
 Introduction
 ============
 
-cuML accelerates machine learning on GPUs. The library follows a
-couple of key principles, and understanding these will help you take
-full advantage cuML.
-
+cuML's design is built around three core principles that work together to make
+GPU-accelerated machine learning both powerful and accessible. These principles
```

Comment:
I read this and was expecting to read the principles next. I think the next three headings are the principles, but I think we could list them explicitly here to hammer home the point/make sure readers know what they are